debug.go

```diff
@@ -67,7 +69,7 @@ func startDebugServer(config sequinsConfig) {
 
 	if config.Debug.Expvars {
 		mux.HandleFunc("/debug/vars", expvarHandler)
-		expStats = newStats(config.LocalStore)
+		expStats = newStats(config.LocalStore, sq.peers.ShardID)
```

Comment:
Does this not cause problems when sequins is run locally? I think `sq.peers` can be `nil` if sharding isn't on so we might have to guard against that case.